common/config-manager.h

```diff
@@ -60,38 +60,44 @@ class ConfigManager : public Singleton<ConfigManager> {
 	private:
 		StringMap _entries;
 		StringMap _keyValueComments;
-		String _domainComment;
+		String    _domainComment;
 
 	public:
 		typedef StringMap::const_iterator const_iterator;
-		const_iterator begin() const { return _entries.begin(); }
-		const_iterator end()   const { return _entries.end(); }
+		const_iterator begin() const { return _entries.begin(); } /*!< Return the beginning position of configuration entries. */
+		const_iterator end()   const { return _entries.end(); }   /*!< Return the ending position of configuration entries. */
 
-		bool empty() const { return _entries.empty(); }
+		bool           empty() const { return _entries.empty(); } /*!< Return the empty configuration entries. */
 
-		bool contains(const String &key) const { return _entries.contains(key); }
+		bool           contains(const String &key) const { return _entries.contains(key); } /*!< Check whether the domain contains a @p key. */
 
-		String &operator[](const String &key) { return _entries[key]; }
-		const String &operator[](const String &key) const { return _entries[key]; }
+		String        &operator[](const String &key) { return _entries[key]; } /*!< Return all entries containing a @p key. */
+		const String  &operator[](const String &key) const { return _entries[key]; } /*!< @overload */
 
-		void setVal(const String &key, const String &value) { _entries.setVal(key, value); }
+		void           setVal(const String &key, const String &value) { _entries.setVal(key, value); } /*!< Assign a @p value to a @p key. */
 
-		String &getVal(const String &key) { return _entries.getVal(key); }
-		const String &getVal(const String &key) const { return _entries.getVal(key); }
-		bool tryGetVal(const String &key, String &out) const { return _entries.tryGetVal(key, out); }
+		String        &getVal(const String &key) { return _entries.getVal(key); } /*!< Retrieve the value of a @p key. */
+		const String  &getVal(const String &key) const { return _entries.getVal(key); } /*!< @overload */
+         /**
```

Comment:
@henke37  you added this recently, can you let me know what is the right description?

Reply:
It retrieves the value if the key exists and leaves the referenced variable unchanged if the key does not exist.

Reply:
thanks